modules/openy_features/openy_block/modules/openy_block_featured_highlights/config/install/field.storage.block_content.field_block_content.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - block_content
+    - text
+id: block_content.field_block_content
+field_name: field_block_content
+entity_type: block_content
+type: text_long
+settings: {  }
+module: text
+locked: false
+cardinality: 1
+translatable: true
+indexes: {  }
+persist_with_no_fields: false
```

Comment:
true?
@hamrant if this option enabled, can we have storage persistent? In this case no ned for openy_prgf_storage trick at all

Reply:
@podarok interesting, I will check this before implementing `openy_prgf_storage`